login-form/src/components/page-components/Dashboard.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import { Box } from "rebass";
+
+import { getDashboardData } from "../../redux/actions";
+import { RebassHeading } from "../ui-components/RebassHeading";
+import { RebassLabel } from "../ui-components/RebassLabel";
+import { LoadingScreen } from "./LoadingScreen";
+
+const Dashboard = ({
+  getDashboard,
+  email,
+  name,
+  origin,
+  isLoading,
+  dashBoardHeader,
+}) => {
+  useEffect(() => {
+    const user = getDashboard();
+    console.log("NEWUSER", user);
+  }, []);
+
+  return (
+    <Box m={[2, 3, 4]}>
+      <RebassHeading
+        name={dashBoardHeader}
+        fontSize={[5, 6, 7]}
+        textAlign={"center"}
+      />
+
+      <Box
+        bg="#E8E8E8"
+        mt={[2, 3, 4]}
+        mb={[2, 3, 4]}
+        p={[2, 3, 4]}
+        sx={{ borderWidth: "1px", borderStyle: "solid", borderColor: "black" }}
+      >
+        <RebassHeading
+          name={"Information"}
+          fontSize={[3, 4, 5]}
+          marginBottom={[2, 3, 4]}
+          marginTop={[2, 3, 4]}
+        />
+        {!isLoading ? (
+          <RebassLabel
+            fontSize={[2, 3, 4]}
+            fontWeight="bold"
+            fontFamily="serif"
+            name={`
+          Email address : ${email} 
+          Name : ${name}
+         Origin Port : ${origin}`}
+          />
+        ) : (
+          ("isLoading...", (<LoadingScreen />))
+        )}
+      </Box>
+    </Box>
+  );
+};
+
+const mapStateToProps = (state) => {
```

Comment:
you can destructure `getDashboard` so no need to duplicate the state every time. You can use the same pattern in other places also.

```suggestion
const mapStateToProps = ({ getDashboard }) => ({
  email: getDashboard.email
})
```